NuRadioReco/modules/LOFAR/beamformingDirectionFitter_LOFAR.py

```diff
@@ -89,40 +68,49 @@ def _direction_fit(self, station, ant_positions):
         dominant_pol = np.argmax(np.bincount(dominant_pol_traces))
         self.logger.debug(f"Dominant polarisation is {dominant_pol}")
 
-        # Collect the Efield traces - the e-field from the converter has eR as [0] component -> do +1 in index
-        fft_traces = np.array([trace.get_frequency_spectrum()[dominant_pol + 1]
+        # Collect the Efield traces
+        fft_traces = np.array([trace.get_frequency_spectrum()[dominant_pol]
                                for trace in station.get_electric_fields()])
 
         def negative_beamed_signal(direction):
             theta = direction[0]
             phi = direction[1]
-            direction_cartesian = hp.spherical_to_cartesian(theta, phi)
-
-            delays = geometric_delays(ant_positions, direction_cartesian)
-
-            out = beamformer(fft_traces, freq, delays)
-            timeseries = fft.freq2time(out, 200 * units.MHz)  # TODO: is this really necessary?
+            my_direction_cartesian = hp.spherical_to_cartesian(theta, phi)
```

Comment:
I am not sure, I like variables with my_ ... this sounds like a hack (without having tested this).

Reply:
I am not sure what you mean?